README.md

```diff
@@ -1,25 +1,94 @@
-# The PHP framework that gets out of your way.
-[![Coverage Status](https://coveralls.io/repos/github/tempestphp/tempest-framework/badge.svg?branch=main)](https://coveralls.io/github/tempestphp/tempest-framework?branch=main)
-
-Read how to get started with Tempest [here](https://github.com/tempestphp/tempest-docs/blob/master/01-getting-started.md).
-
-Zero config, zero overhead. This is Tempest:
-
-```php
-final readonly class BookController
-{
-    #[Get('/blog')]
-    public function index() { /* … */ }
-    
-    #[Get('/blog/{post}')]
-    public function show(Post $post) { /* … */ }
-}
-
-final readonly class RssSyncCommand
-{
-    public function __construct(private Console $console) {}
-
-    #[ConsoleCommand('rss:sync')]
-    public function __invoke(bool $force = false)  { /* … */ }
-}
-```
\ No newline at end of file
+<p align="center">
+  <a href="https://github.com/tempestphp/tempest-framework" target="_blank">
+
+![](.github/tempest-logo.svg)
+  </a>
+</p>
+<p align="center">
+    <a href="LICENSE" target="_blank">
+        <img alt="Software License" src="https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square">
+    </a> 
+    <a href="https://coveralls.io/github/tempestphp/tempest-framework?branch=main" target="_blank">
+        <img alt="Coverage Status" src="https://coveralls.io/repos/github/tempestphp/tempest-framework/badge.svg?branch=main">
+    </a>
+</p>
+
+[TempestPHP](https://github.com/tempestphp/tempest-framework) is a PHP MVC micro-framework that gets out of your way. Its design philosophy is that developers should write as little framework-related code as possible, so that they can focus on application code instead. Our primary goal is to provide a structured framework that enables PHP users at all levels to rapidly develop robust web applications, without any loss to flexibility.
+
+
+## Installing TempestPHP via Composer
```

Comment:
I'm a bit confused, why four ways of writing Tempest?

- TEMPEST
- TempestPHP
- tempestphp
- Tempest

The framework's name is "Tempest"